docker/oap/Dockerfile.oap

```diff
@@ -14,28 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE='adoptopenjdk/openjdk11:alpine'
+ARG BASE_IMAGE='adoptopenjdk/openjdk11:alpine-jre'
 
-FROM golang:1.14 AS cli
-
-ARG COMMIT_HASH=9f267876493943716434fdaa30047a14c0b5b2d9
-ARG CLI_CODE=${COMMIT_HASH}.tar.gz
-ARG CLI_CODE_URL=https://github.com/apache/skywalking-cli/archive/${CLI_CODE}
-
-ENV CGO_ENABLED=0
-ENV GO111MODULE=on
-
-WORKDIR /cli
-
-ADD ${CLI_CODE_URL} .
-RUN tar -xf ${CLI_CODE} --strip 1
-RUN rm ${CLI_CODE}
-
-RUN mkdir -p /skywalking/bin/
-RUN make linux && mv bin/swctl-latest-linux-amd64 /skywalking/bin/swctl
+FROM apache/skywalking-base:8.7.0-es7 AS cli
```

Comment:
Although this works for now, but when we begin to adopt infra-e2e (https://github.com/apache/skywalking/issues/7368) I think it's highly possible we will have to enhance CLI and then compile from source codes here, so what about just updating the `COMMIT_HASH` at line 21?

Reply:
> Although this works for now, but when we begin to adopt infra-e2e (#7368) I think it's highly possible we will have to enhance CLI and then compile from source codes here, so what about just updating the `COMMIT_HASH` at line 21?

Well ignore this, the CLI in infra-e2e should be run outside of OAP container, so the cane here is OK to me

Reply:
Thanks for sharing this, I will update commit sha instead of switching back to released images to get CLI.